contracts/optimize/SlotHelper.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// 存储方法
+library SlotHelper{
+    uint internal constant RIGHTSLOTDATASHIFT = 32;
```

Comment:
uint => uint256
_ should be used to separate words

Reply:
https://github.com/QuarkChain/evm-large-storage/pull/1/commits/75f580a23cb73d6252bdd604650d3edf3d73c4f6